nativelib/src/main/resources/scala-native/gc/none/gc.c

```diff
@@ -1,30 +1,83 @@
 #include <stdlib.h>
 #include <sys/mman.h>
-
+#include "Parsing.h"
+#include "../commix/Memory.h"
```

Comment:
Location and name of this file has changed after adding Windows support (since it is not case-sensitived and clashed with other header files)
```suggestion
#include "MemoryInfo.h"
```